lib/build/cpp/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG BUILD_BASE_IMG_TAG
```

Comment:
To build this, run `docker build . -f Dockerfile.cpp --build-arg BUILD_BASE_IMG_TAG=0.2.0-master0076 -t symbolica/build-cpp:dev`